packages/front-end/components/Radix/RadioGroup.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactElement } from "react";
+import { Flex, Text, RadioGroup as RadixRadioGroup } from "@radix-ui/themes";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+import HelperText, { getRadixColor } from "@/components/Radix/HelperText";
+
+export type RadioOptions = {
+  value: string;
+  label?: string | ReactElement;
+  description?: string | ReactElement;
+  sub?: string;
```

Comment:
The design system does not support this additional level of text.  I know this is used today in the metric type selector.  We should check with Kari about how we want to handle that.